src/emulator/discord.cpp

```diff
@@ -0,0 +1,35 @@
+#ifdef NDEBUG
+#include <discord.h>
+#include <discord_rpc.h>
+#include <discord_register.h>
+
+namespace discord
+{
+	void initialize(const std::string& application_id)
+	{
+		DiscordEventHandlers handlers = {};
+		Discord_Initialize(application_id.c_str(), &handlers, 1, NULL);
+	}
+
+	void shutdown()
+	{
+		Discord_Shutdown();
+	}
+
+	void update_presence(const std::string& state, const std::string& details, bool reset_timer)
+	{
+		DiscordRichPresence discordPresence = {};
```

Comment:
Remember to adapt our style... Shouldn't this be discord_presence? etc.

Reply:
copy-pasta, forgot about it.
also, should i credit rpcs3 for this?

Reply:
I did the same for my GDB stub. Go ahead if you'd like :)